### PR TITLE
opensaml 2.6.4 orbit

### PR DESCRIPTION
**Pull request by madurangasiriwardena**

### CODE REVIEW

File: opensaml/2.6.4.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+                    <groupId>org.apache.xalan</groupId>
+                    <artifactId>xalan</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.xerces</groupId>
+                    <artifactId>xercesImpl</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
```
Comment (callkalpa) [inline opensaml/2.6.4.wso2v1/pom.xml]: All dependencies should be optional

File: opensaml/2.6.4.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+                            bsh;resolution:=optional,
+                            nu.xom;resolution:=optional,
+                            org.apache.commons.beanutils;resolution:=optional,
+                            org.apache.commons.collections.iterators;resolution:=optional,
+                            org.apache.commons.configuration;resolution:=optional,
+                            org.apache.commons.fileupload.*;resolution:=optional,
+                            org.apache.log4j.*;resolution:=optional,
+                            org.owasp.validator.html;resolution:=optional,
+                        </Import-Package>
+                        <Embed-Dependency>esapi;scope=compile|runtime;inline=false</Embed-Dependency>
```
Comment (callkalpa) [inline opensaml/2.6.4.wso2v1/pom.xml]: Why embedded dependencies?

File: opensaml/2.6.4.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>1.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${pom.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${pom.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.opensaml.*; version="${opensaml.export.version}",
+                            org.joda.* ; version="${joda.export.version}",
```
Comment (callkalpa) [inline opensaml/2.6.4.wso2v1/pom.xml]: Normally the export version is the same as the project version, i.e. ${project.version}. Is there any specific reason as to why we have deviated here?

File: opensaml/2.6.4.wso2v1/pom.xml (hunk 1 above)
Comment (madurangasiriwardena) [inline opensaml/2.6.4.wso2v1/pom.xml]: corrected

File: opensaml/2.6.4.wso2v1/pom.xml (hunk 2 above)
Comment (madurangasiriwardena) [inline opensaml/2.6.4.wso2v1/pom.xml]: refer to https://wso2.org/jira/browse/CARBON-15455

**Pull request merged**